### ISSUE TITLE
WANTED: Additional Cube Grid values.

### ISSUE THREAD

**Issue by NeoSword** [SEModCommunity/SE-Community-Mod-API#117]
It would be very helpful for cleanup if the cube grid entry list would include:

Owner Steam Name (Somewhere near current Owner ID)
IsDead value

Not sure it's possible but would be VERY nice:
Last Server log on date

Thanks for the great tool!

**Comment by chessmaster42** [comment]
Quick point of clarification: cube grids don't have an owner but the blocks do. So that said can you confirm that you want some extra read-only fields on cube blocks to show the name next to the owner id?

**Comment by NeoSword** [comment]
Yes, currently a block will display owner ID. I would like to see an additional value of owner name and if that player ID is dead.
